bpf/Makefile

```diff
@@ -43,3 +41,25 @@ install:
 		done; \
 	done
 	$(MAKE) -C go install
+
+
+else
+
+all:
+
+install:
+	$(INSTALL) -m 0644 -t $(DESTDIR)$(LIBDIR)/cilium/ $(BPF:.o=.c)
+	$(INSTALL) -m 0755 -t $(DESTDIR)$(LIBDIR)/cilium/ $(SCRIPTS)
+	$(INSTALL) -m 0644 -t $(DESTDIR)$(LIBDIR)/cilium/lib/ $(LIB)
+	for dir in `find include probes -type d`; do \
+		$(INSTALL) -m 0755 -d $(DESTDIR)$(LIBDIR)/cilium/$$dir; \
+		for f in `find $$dir -maxdepth 1 -type f`; do \
+			$(INSTALL) -m 0644 -t $(DESTDIR)$(LIBDIR)/cilium/$$dir $$f; \
+		done; \
+	done
```

Comment:
Ah now I understood what you were talking about. The problem was that `$(MAKE) -C go install` was supposed to be run when `PKG_BUILD==""` but not otherwise. I've made a change, is it acceptable like that?